pom.xml

```diff
@@ -106,7 +106,7 @@
     <kafka.clients.version>0.10.2.1</kafka.clients.version>
     <aws.sdk.version>1.11.133</aws.sdk.version>
     <poi.version>3.16</poi.version>
-    <hadoop.version>2.4.0</hadoop.version>
+    <hadoop.version>2.7.3</hadoop.version>
```

Comment:
Better to use HDP version, they backport lots of bugs

Reply:
As discussed not changing this.